extensions/import/src/test/utils.test.ts

```diff
@@ -184,6 +185,9 @@ export class TestButton implements azdata.window.Button {
 }
 
 export class TestExtensionContext implements vscode.ExtensionContext {
+	constructor(extensionPath: string){
+		this.extensionPath = extensionPath;
```

Comment:
I'd suggest moving the code that gets the extension path into this class instead (make a separate init function probably). That way if other tests want to use it they don't have to reimplement the logic. 

Although I've been thinking about this and I wonder if it'd just be easiest/best to have the context available either as a static var from the extension main class or through the public API.

@aaomidi Do you have any thoughts on that?